.npmrc

```diff
@@ -0,0 +1,2 @@
+ignore-workspace-root-check=true
```

Comment:
viem has 

```
enable-pre-post-scripts=true
lockfile=true
prefer-workspace-packages=true
```
anything else we want from this?

Reply:
No. We don’t have pre and post scripts (also imo they are bad). We don’t have any workspace packages and I believe this is the default anyways. Lock file true I do t know what it means

Side note unrelated, viems npmrc is stale because none of these options do anything with bun I believe